frontend/src/App.tsx

```diff
@@ -62,7 +62,7 @@ export const SnackBarContext = createContext({
 });
 
 const App = () => {
-  const queryClient = new QueryClient();
+  const [queryClient] = useState(() => new QueryClient());
```

Comment:
This was a core issue that needed to be fixed. The query client kept getting a new instance everytime the App component re-rendered. This way of setting up makes sure it always is the same query client instance.

Reply:
You're right, this looks better